griptape/drivers/prompt_model/bedrock_claude_prompt_model_driver.py

```diff
@@ -0,0 +1,49 @@
+import json
+from attr import define, field, Factory
+from griptape.artifacts import TextArtifact
+from griptape.utils import PromptStack
+from griptape.drivers import BasePromptModelDriver
+from griptape.tokenizers import AnthropicTokenizer
+
+
+@define
+class BedrockClaudePromptModelDriver(BasePromptModelDriver):
+    tokenizer: AnthropicTokenizer = field(
+        default=Factory(
+            lambda: AnthropicTokenizer(model="anthropic.claude-v2"),
+        ),
+        kw_only=True,
+    )
+    top_p: float = field(default=0.999, kw_only=True)
+    top_k: int = field(default=250, kw_only=True)
+
+    def prompt_stack_to_model_input(self, prompt_stack: PromptStack) -> dict:
+        prompt_lines = []
+
+        for i in prompt_stack.inputs:
+            if i.is_assistant():
+                prompt_lines.append(f"Assistant: {i.content}")
+            else:
+                prompt_lines.append(f"Human: {i.content}")
+
+        prompt_lines.append("Assistant:")
+
+        prompt = "\n".join(prompt_lines)
+        return {"prompt": prompt}
+
+    def prompt_stack_to_model_params(self, prompt_stack: PromptStack) -> dict:
+        prompt = self.prompt_driver.prompt_stack_to_string(prompt_stack)
+
+        return {
+            "anthropic_version": "bedrock-2023-05-31",
```

Comment:
Same here: should this be a class-level property?

Reply:
@vasinov I'm actually going to remove it. I think I had captured it from a network request, but it doesn't appear to be a documented parameter.